pynetdicom3/sop_class.py

```diff
@@ -627,7 +627,8 @@ def SCP(self, msg):
 
         # Callback - C-FIND
         try:
-            result = self.AE.on_c_find(dataset)
+            # So that the user knows which SOPClass is used.
+            result = self.AE.on_c_find(dataset, self)
```

Comment:
It might be better to pass just the SOP Class UID rather than the whole ServiceClass object